.github/workflows/sanitizer.yml

```
@@ -60,3 +60,18 @@ jobs:
       - name: UBSAN Test
         run: |
           ./build-ubsan/uv_run_tests_a
+  sanitizers-windows:
+    runs-on: windows-2022
+    steps:
+      - uses: actions/checkout@v2
+      - name: Envinfo
+        run: npx envinfo
+
+      - name: UBSAN Build
+        run: |
+          mkdir build-ubsan
+          cmake -B build-ubsan -G Ninja -DBUILD_TESTING=ON -DUBSAN=ON -DCMAKE_BUILD_TYPE=Debug -DCMAKE_C_COMPILER=clang
```

```suggestion
          cmake -B build-ubsan -G "Visual Studio 17 2022" -DBUILD_TESTING=ON -DUBSAN=ON -DCMAKE_BUILD_TYPE=Debug -DCMAKE_C_COMPILER=clang
```

Why is this added as both a sanitizers and CI-win workload? Do we just need one?

---

One config is testing pure Visual Studio generator, plus MSVC's CL, and the other is testing Ninja generator, with clang as compiler.

The build failure your MR is trying to fix was affecting Ninja generator only, so this suggested change would hide the problem.

Note that using Ninja generator with Visual Studio is pretty common when using it's cmake integration, as it is fully supported and produces much faster builds.

---

Looks like it must be installed explicitly now:
https://github.com/actions/runner-images/issues/8343#issuecomment-1727810519